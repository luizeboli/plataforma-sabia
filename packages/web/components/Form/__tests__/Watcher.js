import React from 'react';
import { render } from 'test-utils';
import Watcher from '../Watcher';

test('it renders the form.watch return', () => {
	const { container } = render(
		<Watcher
			form={{
				watch: () => 50,
				control: {},
			}}
			property="my-field"
			render={(element) => {
				return <div>{element}</div>;
			}}
		/>,
	);

	expect(container).toMatchSnapshot();
});
