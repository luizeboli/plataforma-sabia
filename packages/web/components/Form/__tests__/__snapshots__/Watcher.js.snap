// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the form.watch return 1`] = `
<div>
  <div>
    50
  </div>
</div>
`;
